cord19-explorer/src/main/webapp/WEB-INF/views/paper.html

```diff
@@ -33,6 +33,8 @@ <h3>[[${#strings.defaultString(publication.metadata.title, 'No title available')
    		 <span th:if="${author.affiliation.locationRegion != null}" th:remove="tag">[[${author.affiliation.locationRegion}]],</span> 
    		 <span th:if="${author.affiliation.locationCountry != null}" th:remove="tag">[[${author.affiliation.locationCountry}]]</span>
    		 <span th:if="${author.affiliation.selectedWikiarticle != null}"><span class="label label-info">matched</span> <i>[[${author.affiliation.selectedWikiarticle.title}]]</i> ([[${author.affiliation.selectedWikiarticle.coordinates}]])</span>
+   		 <span th:if="${author.email != null}">[[${author.email}]] </span>
+   		 <span th:if="${author.uri != null}">[[${author.uri}]] </span>
```

Comment:
This page already includes most fields from MetaDataImpl, PersonImpl and AffiliationImpl types.

Reply:
there is a lot more metadata on the publication object (e.g. journal, doi, pmcid, etc)